Info/function.json

```diff
@@ -5,7 +5,7 @@
       "type": "httpTrigger",
       "direction": "in",
       "name": "req",
-      "route": "v1/info",
+      "route": "api/v1/info",
```

Comment:
Not actually sure of this change, afaik api prefix is added by the runtime

Reply:
I set 
`    "http": {
      "routePrefix": ""
    },`

 into `host.json` file. I'm doing some tests with io-mock and it's working.
 
 I think it's up to us to decide that to do. What do you prefer?

Reply:
I did like what we said here https://github.com/pagopa/io-functions-template/pull/21#issuecomment-808039140